README.md

```diff
@@ -1 +1,112 @@
-# [WIP] LAI-API-Access-UI-Component
+<div align="center">
+    <h1>
+        Lightning API Access UI Component
+    </h1>
+    <img src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/logos/lightning-ai.png" width="400px">
+
+<div align="center">
+
+<p align="center" style="color:grey">Enable users to engage with your app via an API</a></p>
+
+______________________________________________________________________
+
+
+<p align="center">
+  <a href="#getting-started">Getting Started</a> •
+  <a href="https://www.lightning.ai/">Lightning AI</a> •
+  <a href="https://lightning.ai/components">Lightning Component Gallery</a>
+</p>
+
+[![Slack](https://img.shields.io/badge/slack-chat-green.svg?logo=slack)](https://www.pytorchlightning.ai/community)
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Lightning-AI/lightning/blob/master/LICENSE)
+
+</div>
+</div>
+
+______________________________________________________________________
+
+
+# API Access Frontend Component
+
+This component allows you to enable users to engage with your app via an API. It is a simple component that allows you to create an UI guide on how to send requests to your App APIs. The component is built with [Lightning's StaticWebFrontend](https://lightning.ai/lightning-docs/api_reference/generated/lightning_app.frontend.web.StaticWebFrontend.html) and [React](https://reactjs.org/) for the interface.
+
+
+## Getting Started
+
+### Installation
+
+```bash
+lightning install component git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main
```

Comment:
Cool, does this actually work? Let's also remember to update it once the component is in the gallery

Reply:
```
❯ lightning install component git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main >> intall_logs.txt
INFO: 
    ⚡ Installing Lightning component ⚡

    component name : LAI-API-Access-UI-Component
    developer      : Lightning-AI

    ⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡
    WARNING: this is NOT an official Lightning Gallery component
    Install at your own risk
    ⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡

    Installation runs the following command for you:

    pip install git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main
    

INFO: ⚡ RUN: pip install
  Running command git clone --filter=blob:none --quiet https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git /private/var/folders/3j/h7yk_n256r15z7ttg578k8lw0000gn/T/pip-req-build-o6fl4y5f

[notice] A new release of pip available: 22.2.2 -> 22.3
[notice] To update, run: pip install --upgrade pip
INFO: 
        ⚡ Installed! ⚡

        to use your component:
        from the_component import TheClass

        make sure to add this entry to your Lightning APP requirements.txt file:
        {git_url}

        if you want to uninstall, run this command:
        pip uninstall {uninstall_step}
```
yup, here are the CLI logs